src/views/SpaceSettings.vue

```diff
@@ -237,7 +237,10 @@ function formatSpace(spaceRaw) {
 watchEffect(async () => {
   if (!props.spaceLoading) {
     try {
-      const uri = await getSpaceUri(props.spaceId);
+      const uri = await getSpaceUri(
+        props.spaceId,
+        import.meta.env.VITE_DEFAULT_NETWORK
```

Comment:
Maybe we can make this optional?

Reply:
What do you mean? It is. The second parameter of `getSpaceUri` deafults to '1' same as `VITE_DEFAULT_NETWORK`. I'm just not 100% sure if `VITE_DEFAULT_NETWORK` is the best choice for this.

Reply:
Oh ok, cool then :) I forgot about the default param on getSpaceUri